Tests/Datagrid/ProxyQueryTest.php

```diff
@@ -45,46 +46,36 @@ public function dataGetFixedQueryBuilder()
      */
     public function testGetFixedQueryBuilder($class, $alias, $id, $expectedId, $value, $identifierType)
     {
-        $meta = $this->getMockBuilder('Doctrine\ORM\Mapping\ClassMetadataInfo')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $meta = $this->createMock('Doctrine\ORM\Mapping\ClassMetadataInfo');
         $meta->expects($this->any())
             ->method('getIdentifierFieldNames')
             ->willReturn(array($id));
         $meta->expects($this->any())
             ->method('getTypeOfField')
             ->willReturn($identifierType);
 
-        $mf = $this->getMockBuilder('Doctrine\ORM\Mapping\ClassMetadataFactory')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $mf = $this->createMock('Doctrine\ORM\Mapping\ClassMetadataFactory');
         $mf->expects($this->any())
             ->method('getMetadataFor')
             ->with($this->equalTo($class))
             ->willReturn($meta);
 
-        $platform = $this->getMockBuilder('Doctrine\DBAL\Platforms\PostgreSqlPlatform')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $platform = $this->createMock('Doctrine\DBAL\Platforms\PostgreSqlPlatform');
 
-        $conn = $this->getMockBuilder('Doctrine\DBAL\Connection')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $conn = $this->createMock('Doctrine\DBAL\Connection');
         $conn->expects($this->any())
             ->method('getDatabasePlatform')
             ->willReturn($platform);
 
-        $em = $this->getMockBuilder('Doctrine\ORM\EntityManager')
-            ->disableOriginalConstructor()
-            ->getMock();
+        $em = $this->createMock('Doctrine\ORM\EntityManager');
         $em->expects($this->any())
             ->method('getMetadataFactory')
             ->willReturn($mf);
         $em->expects($this->any())
             ->method('getConnection')
             ->willReturn($conn);
 
-        $q = $this->getMock('PDOStatement');
+        $q = $this->createMock('PDOStatement');
```

Comment:
I think the error comes from this line here

```
1) Sonata\DoctrineORMAdminBundle\Tests\Datagrid\ProxyQueryTest::testGetFixedQueryBuilder with data set #0 ('aaa', 'bbb', 'id', 'id_idx', 33, 'integer')
PHPUnit_Framework_MockObject_RuntimeException: You cannot serialize or unserialize PDOStatement instances
/home/travis/build/sonata-project/SonataDoctrineORMAdminBundle/Tests/Helpers/PHPUnit_Framework_TestCase.php:56
/home/travis/build/sonata-project/SonataDoctrineORMAdminBundle/Tests/Datagrid/ProxyQueryTest.php:78
```

I found this, what do you think guys? And why is it failing now and not before?
https://github.com/sebastianbergmann/phpunit/issues/1171#issuecomment-37096058